php/Dockerfile.tpl

```diff
@@ -36,8 +37,14 @@ RUN echo "Starting ..." && \
     echo "Starting PHP" && \
     docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ && \
     docker-php-ext-configure intl && \
-    docker-php-ext-install -j$(getconf _NPROCESSORS_ONLN) bcmath exif gd intl pcntl pdo_mysql pdo_pgsql soap sockets zip && \
+    docker-php-ext-install -j$(getconf _NPROCESSORS_ONLN) bcmath exif gd intl pcntl pdo_mysql pdo_pgsql pgsql soap sockets zip && \
     pecl install apcu-${APCU_VERSION} && \
+    pecl install memcached-${MEMCACHED_VERSION} && \
+    docker-php-ext-enable memcached && \
+    docker-php-ext-configure pgsql -with-pgsql=/usr/local/pgsql && \
+    curl -o /tmp/ssh2.tgz https://pecl.php.net/get/ssh2 && \
+    pecl install ssh2 /tmp/ssh2.tgz && \
```

Comment:
`pecl install ssh2` without `curl` before does not work?

Reply:
curl is already installed (line 23).
I check if ssh2 extension exist on my local
@mremi